pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/dsl/expression/Collection.kt

```diff
@@ -17,10 +18,19 @@ fun <T> Expression<Collection<T>>.isEmpty(): Expression<Boolean> =
         operation = BinaryOperation.ValidatePredicate(),
     )
 
+fun <T> Expression<Collection<T>>.size(): Expression<Int> =
+    Expression.UnaryInvoke(value = this, operation = UnaryOperation.SizeOf)
+
+fun Expression<Int>.absoluteValue(): Expression<Int> =
+    Expression.UnaryInvoke(value = this, operation = UnaryOperation.AbsoluteValue)
+
+fun Expression<Kroner>.absoluteKronerValue(): Expression<Kroner> =
```

Comment:
Kan disse to ha samme navn, `absoluteValue`eller fører det til navnekollisjon?

Reply:
Mener man kan bruke `@JvmName` for å spesifisere et navn som ikke overlapper.

Reply:
byttet navn. Fixed.